models/public/retinaface-resnet50-pytorch/README.md

```diff
@@ -0,0 +1,127 @@
+# retinaface-resnet50-pytorch
+
+## Use Case and High-Level Description
+
+The `retinaface-resnet50-pytorch` model is a PyTorch\* implementation of medium size model with ResNet50 backbone for Face Localization. It can output face bounding boxes and five facial landmarks in a single forward pass. More details provided in the [paper](https://arxiv.org/abs/1905.00641) and [repository](https://github.com/biubug6/Pytorch_Retinaface)
+
+## Specification
+
+| Metric                                                        | Value       |
+|---------------------------------------------------------------|-------------|
+| AP ([WIDER](http://mmlab.ie.cuhk.edu.hk/projects/WIDERFace/)) | 91.78%      |
+| GFLOPs                                                        | TBD         |
+| MParams                                                       | TBD         |
+| Source framework                                              | PyTorch\*   |
+
+Average Precision (AP) is defined as an area under the
+[precision/recall](https://en.wikipedia.org/wiki/Precision_and_recall)
+curve. All numbers were evaluated by taking into account only faces bigger than
+64 x 64 pixels.
+
+Accuracy validation approach different from described in the original repo.
```

Comment:
is it possible to briefly describe what are the differences and why they are needed?

Reply:
original Wider evaluation protocol different from validation strategy in AC.
It splits whole validation set on 3 predefined subset (hard, medium, easy) and verify all these sets separately.

Reply:
Done